src/librustc_session/options.rs

```diff
@@ -968,4 +979,6 @@ options! {DebuggingOptions, DebuggingSetter, basic_debugging_options,
         "link the `.rlink` file generated by `-Z no-link`"),
     new_llvm_pass_manager: Option<bool> = (None, parse_opt_bool, [TRACKED],
         "use new LLVM pass manager"),
+    src_hash_algorithm: Option<SourceFileHashAlgorithm> = (None, parse_src_file_hash, [TRACKED],
+        "override cryptographic hash algorithm of source files in debug info (`md5`, or `sha1`)"),
```

Comment:
You use the c word here.

Reply:
Agreed. 'Cryptographic' isn't necessary here.

Reply:
Thanks again! :heart: